### PR TITLE
Listmonk setup

### PR DESCRIPTION
**Pull request by kaydee0502**

### CODE REVIEW

File: devsnest/app/controllers/api/v1/admin/listmonk_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    module Admin
+      # link controller for urls
+      class ListmonkController < ApplicationController
+        include JSONAPI::ActsAsResourceController
+        # before_action :admin_auth
```
Comment (Yug-gurnani) [inline devsnest/app/controllers/api/v1/admin/listmonk_controller.rb:7]: uncomment this

File: devsnest/app/controllers/api/v1/admin/listmonk_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    module Admin
+      # link controller for urls
```
Comment (Yug-gurnani) [inline devsnest/app/controllers/api/v1/admin/listmonk_controller.rb:4]: Link controller?

File: devsnest/app/controllers/concerns/api_render_concern.rb [hunk 3]
```diff
@@ -20,6 +20,10 @@ def render_unauthorized
     api_render 401, code: :unauthorized, errors: ['Unauthorized']
   end
 
+  def render_unprocessable
+    api_render 422, code: :unprocessable_entity, errors: ['Unprocessable Entity']
```
Comment (Yug-gurnani) [inline devsnest/app/controllers/concerns/api_render_concern.rb:24]: we should make the message in errors dynamic

**Review by Yug-gurnani — CHANGES_REQUESTED** [review]
Add Test cases and fix code climate issues :p

File: devsnest/app/controllers/concerns/api_render_concern.rb (hunk 3 above)
Comment (kaydee0502) [reply devsnest/app/controllers/concerns/api_render_concern.rb:24]: Isn't it genric for 422?, and for 401 and 403 given above?

**Review by Yug-gurnani — APPROVED** [review]

**Pull request merged**